### ISSUE TITLE
Suggestion 

### ISSUE THREAD

**Issue by Clappy8** [ryzyx/discordjs-button-pagination#47]
Would love to see a home button on this package for the larger pages and embeds

**Comment by ryzyx** [comment]
not sure if everyone requires that, it'd make this package complex for new users to pass in multiple parameters, not everyone would like a home page button. tho i appreciate ur idea

**Comment by Clappy8** [comment]
yeah true was just an idea that could have been cool for some users but appreciate the reply not many people do reply on here

**Comment by ryzyx** [comment]
yeah man, it was by chance i saw this notif, pretty busy now since school

**Comment by Clappy8** [comment]
ah fair enough
